src/components/dialog/dialog.pw.tsx

```diff
@@ -270,6 +270,7 @@ test.describe("Testing Dialog component properties", () => {
     const firstTextbox = page.getByLabel("Textbox1");
     const closeButton = page.getByLabel("Close");
 
+    await page.waitForTimeout(1000);
```

Comment:
Why are these timeouts needed? 1 second seems like a long time.

Reply:
Each of the dialog tests that either tabs or shift+tabs through the elements regularly fails without some kind of wait before the first tab, especially when running through the UI. I'm not a fan of putting in arbitrary wait times, but this at least makes the test do what it's supposed to. I've reduced each wait time to a quarter of a second.